.github/workflows/check-image-vulnerabilities.yml

```diff
@@ -32,17 +26,51 @@ jobs:
           } >> "$GITHUB_STEP_SUMMARY"
 
   check-image-vulnerabilities:
-    name: Checks images in version ${{ inputs.version }} for vulnerabilities
+    name: Check images in version ${{ inputs.version }} for vulnerabilities
     runs-on: ubuntu-latest
     env:
-      QUAY_BEARER_TOKEN: ${{ secrets.QUAY_RHACS_ENG_VULNERABILITY_BEARER_TOKEN }}
+      # ACS-CS dogfooding instance.
+      ROX_ENDPOINT: "acs-cii18grublkv81uil8gg.acs.rhcloud.com"
```

Comment:
Why put this as a secret? Configuring this on repo level just makes it less visible and a hassle to change because it requires repo admin permissions to view and change. For example I have no way of confirming what instance `STACKROX_CI_INSTANCE_CENTRAL_HOST` is.

Reply:
I support using a secret for this to avoid exposing the URL. 

`STACKROX_CI_INSTANCE_CENTRAL_HOST` is currently set to `disabled`. I'll explain in another comment.

Reply:
I think we should use an environment variable instead of a secret, but agree with not hardcoding it.

Reply:
Set it to organization variable.